benchmarks/orc/README.md

```diff
@@ -0,0 +1,12 @@
+## Orc Benchmarks
+The following four scripts are used to benchmark Orc reading and writing against
+Tuplex's CSV implementation:
+1. `runcsv-read-single.py`
+2. `runcsv-write.py`
+3. `runorc-read-single.py`
+4. `runorc-write.py`
+
+## Usage
+In order to run the benchmarks, you must first import a CSV and Orc file
```

Comment:
How do I import a CSV / ORC file? which path etc. does this need to get placed? What do I need to invoke?